server/src/repositories/metadata.repository.ts

```diff
@@ -29,7 +29,7 @@ export class MetadataRepository implements IMetadataRepository {
         ...DefaultReadTaskOptions,
 
         // Enable exiftool LFS to parse metadata for files larger than 2GB.
-        optionalArgs: ['-api', 'largefilesupport=1'],
+        // optionalArgs: ['-api', 'largefilesupport=1'],
```

Comment:
I still need to look into this one

Reply:
Ok, got a fix as per https://photostructure.github.io/exiftool-vendored.js/interfaces/ExifToolOptions.html#readArgs

Reply:
Fixed by https://github.com/immich-app/immich/pull/6455/commits/cda160fb24aa7b1676ad3743d502b7e8fdeb40a9